lib/dor/workflow/client/queues.rb

```diff
@@ -11,53 +11,70 @@ def initialize(requestor:)
         #   meaning you will get all queued workflows
         # @option opts [Integer] :limit sets the maximum number of workflow steps that can be returned.  Defaults to no limit
         # @return [Array[Hash]] each Hash represents a workflow step.  It will have the following keys:
         #  :workflow, :step, :druid, :lane_id
-        def stale_queued_workflows(repository, opts = {})
-          uri_string = build_queued_uri(repository, opts)
+        def stale_queued_workflows(*args)
+          if args.count == 2
+            Deprecation.warn(
+              self,
+              '`#stale_queued_workflows` only takes one arg: a hash. This will raise an exception in Dor::Workflow::Client 4.0.0'
+            )
+          end
+          uri_string = build_queued_uri(args.last)
           parse_queued_workflows_response requestor.request(uri_string)
         end
 
         # Returns a count of workflow steps that have a status of 'queued' that have a last-updated timestamp older than the number of hours passed in
-        # @param [String] repository name of the repository you want to query, like 'dor' or 'sdr'
+        # @param [String] repository -- deprecated, ignored by workflow service
         # @param [Hash] opts optional values for query
         # @option opts [Integer] :hours_ago steps older than this value will be returned by the query.  If not passed in, the service defaults to 0 hours,
         #   meaning you will get all queued workflows
         # @return [Integer] number of stale, queued steps if the :count_only option was set to true
-        def count_stale_queued_workflows(repository, opts = {})
-          uri_string = build_queued_uri(repository, opts) + '&count-only=true'
+        def count_stale_queued_workflows(*args)
+          if args.count == 2
+            Deprecation.warn(
+              self,
+              '`#count_stale_queued_workflows` only takes one arg: a hash. This will raise an exception in Dor::Workflow::Client 4.0.0'
+            )
+          end
+          uri_string = build_queued_uri(args.last) + '&count-only=true'
           doc = Nokogiri::XML(requestor.request(uri_string))
           doc.at_xpath('/objects/@count').value.to_i
         end
 
         # Returns a list of druids from the workflow service that meet the criteria
         # of the passed in completed and waiting params
         #
-        # @param [Array<String>, String] completed An array or single String of the completed steps, should use the qualified format: `repository:workflow:step-name`
+        # @param [Array<String>, String] completed An array or single String of the completed steps, should use the qualified format: `workflow:step-name`
         # @param [String] waiting name of the waiting step
-        # @param [String] repository default repository to use if it isn't passed in the qualified-step-name
+        # @param [String] repository -- deprecated, ignored by workflow service
```

Comment:
This isn't a parameter that this method accepts. Can it be removed?

Reply:
Cleaned up.